Root/stylesheet.css

```diff
@@ -29,8 +38,15 @@ h1 {
   width: 1000px;
 }
 
+img {
```

Comment:
Since you might have multiple images on the site, it would perhaps be best to give the image a class like "main-img" otherwise these styles will apply to all images.